plugins/janus_streaming.c

```diff
@@ -1589,7 +1730,7 @@ struct janus_plugin_result *janus_streaming_handle_message(janus_plugin_session
 			mp->pin = g_strdup(json_string_value(pin));
 		if(save) {
 			/* This mountpoint is permanent: save to the configuration file too
-			 * FIXME: We should check if anything fails... */
+			* FIXME: We should check if anything fails... */
```

Comment:
Probably an artifact of rebasing & resolving a merge conflict as the guts of this patch are roughly a month old now. ;)